src/olympia/landfill/management/commands/fetch_prod_addons.py

```diff
@@ -25,6 +25,10 @@ def add_arguments(self, parser):
             '--max', metavar='MAX', type=int,
```

Comment:
why is this `MAX` and later it's `options.get('max')`?

Reply:
Good question :) I don't know, it's copied from EnTeQuAk's script, but I'll fix it and see if it still works :)